front/pages/solutions/customer-support.tsx

```diff
@@ -42,11 +42,14 @@ export const getServerSideProps = makeGetServerSidePropsRequirementsWrapper({
   }
 
   return {
-    props: { gaTrackingId: config.getGaTrackingId(), shape: 9 },
+    props: {
+      gaTrackingId: config.getGaTrackingId(),
+      shape: getParticleShapeIndexByName(shapeNames.octahedron),
+    },
   };
 });
 
-export default function CustomerSupport() {
+export default function Content() {
```

Comment:
Rename the file if you change the component name.

Reply:
If I do so, they will all be named "Content". Thought it was better that they all shared the same structure, but happy to revert to older naming.